src/components/ConvincedButton/index.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from "react";
+
+import VideoModal from "../VideoModal";
+
+import styles from "./styles.module.scss";
+
+export default function ConvincedButton() {
+  const [showModal, setShowModal] = useState(false);
+
+  const toggleModal = () => {
+    setShowModal(state => !state);
+  };
+
+  const closeModal = () => {
+    setShowModal(false);
+  };
+
+  return (
+    <div className={styles.ctaSecondary}>
+      <a
+        href="#"
+        title="See how climate change happens"
+        onClick={() => toggleModal()}
```

Comment:
It might be worth using a button here instead of a link, that way, we don't add a `#` to the URL